bin/elixir.ps1

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env pwsh
+
+$MinPowerShellVersion = [version]"7.2.0"
+
+if ($MinPowerShellVersion.CompareTo([version]$PSVersionTable.PSVersion) -eq 1) {
+  Write-Error "This script requires PowerShell version 7.2 or above. Running on $($PSVersionTable.PSVersion)"
+}
+
+$ELIXIR_VERSION = "1.17.0-dev"
+$ScriptPath = Split-Path -Parent $PSCommandPath
+$ErlExec = "erl"
+
+function PrintElixirHelp {
+  $ScriptName = Split-Path -Leaf $PSCommandPath
+  $Help = @"
+Usage: $ScriptName [options] [.exs file] [data]
+
+## General options
+
+  -e "COMMAND"                 Evaluates the given command (*)
+  -h, --help                   Prints this message (standalone)
+  -r "FILE"                    Requires the given files/patterns (*)
+  -S SCRIPT                    Finds and executes the given script in \$PATH
```

Comment:
`\$PATH` should be 
```
`$PATH
```
if you want to escape the $.  But why are using Unix style environment variables?

Reply:
So far we have kept all documentation "consistent" and used `$PATH` to refer to env vars. Perhaps something to reassess but probably better in a separate PR.